Xamarin.Forms.Platform.iOS/Renderers/PageRenderer.cs

```diff
@@ -364,7 +364,9 @@ public override void TraitCollectionDidChange(UITraitCollection previousTraitCol
 		{
 			base.TraitCollectionDidChange(previousTraitCollection);
 
-			if (Forms.IsiOS13OrNewer && previousTraitCollection.UserInterfaceStyle != TraitCollection.UserInterfaceStyle)
+			if (Forms.IsiOS13OrNewer &&
+				previousTraitCollection.UserInterfaceStyle != TraitCollection.UserInterfaceStyle &&
+				UIApplication.SharedApplication.ApplicationState != UIApplicationState.Background)
```

Comment:
This is stopping ios appliaction manager from showing the theme changes until the app is brought back to foreground by the user. It used to show the theme changes because when it went to background ios calls the theme change twice on the app to take a static screenshot of both light and dark theme ready for use in the application manager. Suggest removing UIApplication.SharedApplication.ApplicationState != UIApplicationState.Background from the if statement, this should rectify the problem.